content/static/kitchen-sink/javascripts/kitchen-sink.js

```diff
@@ -547,7 +547,7 @@ $(document)
     // tab
     $('.tab.demo .menu .item')
       .tab({
-        history: true,
+     // history: true,
```

Comment:
I do not quite understand the "history" argument, either. In the documentation, it is called hash history: For static sites, or sites that only require simple history, in-page links can be used to trigger different local tab states. I do not understand what he is saying here. Maybe for simple static sites, history argument is not necessary.